controllers/storagecluster/job_templates_test.go

```diff
@@ -0,0 +1,59 @@
+package storagecluster
+
+import (
+	"context"
+	"testing"
+
+	configv1 "github.com/openshift/api/config/v1"
+	openshiftv1 "github.com/openshift/api/template/v1"
+	api "github.com/openshift/ocs-operator/api/v1"
+	"github.com/stretchr/testify/assert"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"sigs.k8s.io/controller-runtime/pkg/reconcile"
+)
+
+//assert(check if th)
```

Comment:
What's this comment about?

Reply:
Removed. Mindless commenting left unchecked.